playbook/app/pb_kits/playbook/pb_button/button.test.js

```diff
@@ -0,0 +1,91 @@
+
+import React from 'react'
+import { appendAlert, fireEvent, render, screen, waitFor } from '../utilities/test-utils'
+
+import Button from './_button'
+
+// Primary Test Variables
+const htmlType = 'submit',
+  text = 'Button Text',
```

Comment:
If you end up updating this PR, I'd align the const names here. Not required tho.

Reply:
That'd be my preference too but for whatever reason the linter just wants two space indentation instead of lining up the variable names.